Cargo.toml

```diff
@@ -34,10 +34,10 @@ heck = "0.4.0"
 
 [dependencies]
 edgeql-parser = {git = "https://github.com/edgedb/edgedb"}
-edgedb-protocol = {git = "https://github.com/edgedb/edgedb-rust/", features=["all-types"]}
-edgedb-derive = {git = "https://github.com/edgedb/edgedb-rust/"}
-edgedb-errors = {git = "https://github.com/edgedb/edgedb-rust/"}
-edgedb-tokio = {git = "https://github.com/edgedb/edgedb-rust/", features=["admin_socket", "unstable"]}
+edgedb-protocol = {git = "https://github.com/edgedb/edgedb-rust/", rev="dd0f4c945e22da4ef5ccc21016fb8ada2b40e398", features=["all-types"]}
+edgedb-derive = {git = "https://github.com/edgedb/edgedb-rust/", rev="dd0f4c945e22da4ef5ccc21016fb8ada2b40e398"}
+edgedb-errors = {git = "https://github.com/edgedb/edgedb-rust/", rev="dd0f4c945e22da4ef5ccc21016fb8ada2b40e398"}
+edgedb-tokio = {git = "https://github.com/edgedb/edgedb-rust/", rev="dd0f4c945e22da4ef5ccc21016fb8ada2b40e398", features=["admin_socket", "unstable"]}
```

Comment:
I pinned the commit here, as it wasn't building for me with the latest master, but I'm not that familiar with rust/cargo so I may have been doing something wrong.

Reply:
Yeah that looks like a recent upstream breaking change or something. Revert both `Cargo.toml` and `Cargo.lock` and it builds for me.

Reply:
I've updated edgedb-cli dependencies, which should resolve your problem.

I suggest you first merge changes into edgedb-rust and then remove these `rev`s here and run `cargo update`. This will update the lock file to pick up latest changes from edgedb-rust.

Reply:
Thanks, I can confirm that it builds for me with that PR.